lib/cmd_type.py

```diff
@@ -20,7 +20,8 @@
 CMD_LOAD_RESPONSE = 5
 CMD_STORE_REQUEST = 6
 CMD_CONST         = 7
-
+NUM_OPTS = 64
```

Comment:
Sorry, Yufei, lol, I meant `NUM_CMDS = 6` in this file, while `NUM_OPTS = 64` in the `opt_type.py`.